webpack.config.js

```diff
@@ -24,7 +24,8 @@ const config = {
       "fs": false,
       "path": require.resolve("path-browserify"),
       "stream": require.resolve("stream-browserify"),
-      'util': false
+      'util': false,
+      "os": require.resolve("os-browserify")
```

Comment:
I've never messed around with webpack before, but this is close to what the CLI suggested when it threw an error at me (I think the actual require was `os-browserify/browser` but I followed what was already in the file)

Reply:
If it works- great. Seems reasonable. (does anybody truly understand Webpack? 😅😅)